CefSharp.BrowserSubprocess/CefSharp.BrowserSubprocess.csproj

```diff
@@ -111,7 +111,9 @@
     <PostBuildEvent>if $(PlatformName) == x86 (
   call "$(DevEnvDir)..\tools\vsvars32.bat"
   editbin /largeaddressaware $(TargetPath)
-)</PostBuildEvent>
+)
+call "$(DevEnvDir)..\tools\vsvars32.bat"
```

Comment:
I think it's probably safer to call `vsvars64.bat` when building the `x64` version.

So something like below?

```
if $(PlatformName) == x86 (
  call "$(DevEnvDir)..\tools\vsvars32.bat"
  editbin /largeaddressaware /TSAWARE $(TargetPath)
) else (
  call "$(DevEnvDir)..\tools\vsvars64.bat"
  editbin /TSAWARE $(TargetPath)
)
```